src/assets/styles/pikaday_custom.scss

```diff
@@ -0,0 +1,55 @@
+@import "node_modules/pikaday/scss/pikaday.scss";
+
+$pd-day-hover-color: lighten($pd-day-color, 8%);
+
+// @Overwritten styles
+#calendar-container {
+  height: 215px;
```

Comment:
If it's possible to use ems/percentages/etc vs fixed pixels for sizes I think that would be ideal, as the rest of this project is doing that and it is more flexible. BUT I see this is kind of just forking the pikday styles and making adjustments, I have no idea how well that'd all work here. For the height, no big deal, fixed pixel it. If it's easy to swap in ems for margin/padding values though, that would be sweet. (There is also an `$offset` variable defined, that might be handy).